### PR TITLE
Changing English to Hindi Translation

### PR DESCRIPTION
**Pull request by GodWin1100**
# Shithil

```js
const profile=(name)=>{ changed}
```

Consent issue https://github.com/shetty-shithil/react-youtube-clone/issues/1

### CODE REVIEW

**Comment by GodWin1100** [comment]
hi

File: src/components/layout/Header.js [hunk 1]
```diff
@@ -147,7 +147,7 @@ const Header = (props) => {
         <IconButton aria-label="create videos" color="inherit">
           <VideoCallIcon />
         </IconButton>
-        <p>Create</p>
+        <p>Crate</p>
```
Comment (shetty-shithil) [inline src/components/layout/Header.js:150]: There's a spelling mistake.

File: src/components/layout/Header.js [hunk 2]
```diff
@@ -159,7 +159,7 @@ const Header = (props) => {
         >
           <AccountCircle />
         </IconButton>
-        <p>Profile</p>
+        <p>Procter</p>
```
Comment (shetty-shithil) [inline src/components/layout/Header.js:162]:
```suggestion
        <p>Profile</p>
```

**Comment by GodWin1100** [comment]
Trial done

**Pull request closed without merging**